### DISCUSSION TITLE
Css and Ajax not loading in deployment at PythonAnywhere

### DISCUSSION THREAD

**Discussion by Teut2711** [encode/django-rest-framework#8853] [General]
Hello everyone,
I deployed [http://teut.pythonanywhere.com/](this) website on Python Anywhere but somehow the css and js is not loading and the page is plain html. Please help.

**Comment by 2ykwang** [comment]
@Teut2711 

This seems to be an issue unrelated to DRF.

It looks like an infrastructure related issue. The documentation below may help.

 https://help.pythonanywhere.com/pages/DjangoStaticFiles/
